docs/portal_config.md

```diff
@@ -78,6 +79,8 @@ Below is an example, with inline comments describing what each JSON block config
           "label": "Run analysis"
         }
       ],
+      "homepageChartNodesExcludeFiles": true, // optional; specifies whether to exclude File nodes from the homepage chart. Defaults to false
```

Comment:
we need to add `chartNodesExcludeFiles` to the `graphql` section too

Reply:
it is there https://github.com/uc-cdis/data-portal/pull/1208/files#diff-b44409708be4e154876122765a9fd3ada56184bb299149d53f7227a5db35c2eaR46

Reply:
oops, sorry